pkg/chunkenc/unordered_test.go

```diff
@@ -332,13 +375,15 @@ func BenchmarkHeadBlockWrites(b *testing.B) {
 			if tc.unorderedWrites {
 				ts := rnd.Int63()
 				writes = append(writes, entry{
-					t: ts,
-					s: fmt.Sprint("line:", ts),
+					t:                ts,
+					s:                fmt.Sprint("line:", ts),
+					nonIndexedLabels: labels.Labels{{Name: "foo", Value: fmt.Sprint(ts)}},
```

Comment:
It would be good to add it as a separate tc

Reply:
Done